village/src/components/SmurfForm.js

```diff
@@ -14,11 +16,20 @@ class SmurfForm extends Component {
     event.preventDefault();
     // add code to create the smurf using the api
 
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
+    axios
+      .post(`${this.props.baseUrl}/smurfs`, this.state)
+      .then(res => {
+        this.setState({
+          name: "",
+          age: "",
+          height: ""
+        })
+        { this.props.addSmurfs() }
```

Comment:
It looks like this line does not do anything. `addSmurfs()` is not present in the props object.